README.md

```diff
@@ -15,6 +15,7 @@ This project is maintained by
 - Patrick
 - Ye Qin
 - Sean
+-Cesare Corazza
```

Comment:
put a space between the `-` and `Cesare` otherwise it won't render properly as a bullet point in markdown

kthx